tester_app/app/assets/stylesheets/_bootstrap-custom.scss

```diff
@@ -0,0 +1,55 @@
+// This file is used instead of
+// @import 'boostrap';
+// To allow for customization of what parts of bootstrap are imported.
+// Refer to this file: https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/_bootstrap.scss
+
+// Core variables and mixins
+@import 'bootstrap/variables';
+@import 'bootstrap/mixins';
```

Comment:
Actually, all the sass variables are defined with !default: https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/bootstrap/_variables.scss

Sol we could keep this file intact and put our pre- and post- boostrap files outside this and only reference in the application.css.scss. That's probably better.

We should add a comment to the effect that you can customize the variables before bootstrap variables are set with that URL reference.